process/setup_sp.py

```diff
@@ -373,6 +373,100 @@ def process_distant_nodes(
     return(distant_nodes)
 
 
+#Cumulative opportunities (binary)
+#1 if d <= access_dist
+#0 if d > access_dist
+def binary_access_score(df, new_name, old_name, access_dist):
+    """
+    Calculate accessibiity score using binary measure: 1 if access <= access_dist, 0 otherwise
+
+    Parameters
+    ----------
+    df: DataFrame
+        DataFrame with origin-destination distances
+    new_name: list
+        list of new field names for new access score
+    old_name: list
+         list of original field names for distances
+    access_dist: int
+        access distance threshold
+
+    Returns
+    -------
+    DataFrame
+    """
+    df[new_name] = (df[old_name] <= access_dist).astype("Int64").fillna(0)
+    return df[new_name]
+
+
+#Soft threshold access score
+#Higgs, C., Badland, H., Simons, K. et al. The Urban Liveability Index
+def soft_access_score(df, new_name, old_name, access_dist, k):
+    """
+    Calculate accessibiity score using soft threshold approach:
+    1 / (1+ e ^(k *((dist-access_dist)/access_dist)))
+
+    Parameters
+    ----------
+    df: DataFrame
+        DataFrame with origin-destination distances
+    new_name: list
+        list of new field names for new access score columns
+    old_name: list
+         list of original field names for distance columns
+    access_dist: int
+        access distance threshold
+    k: int
+        the slope of decay
+
+    Returns
+    -------
+    DataFrame
+    """
+    df[new_name] = 1 / (1+numpy.exp(k * ((df[old_name]-access_dist) / access_dist)))
+
+    df[new_name] = df[new_name].astype(float).fillna(0)
+    return df[new_name]
+
+#Cumulative-Gaussian
+#Reference: Vale, D. S., & Pereira, M. (2017).
+#The influence of the impedance function on gravity-based pedestrian accessibility measures
+def Cumulative_Gaussian_access_score(df, new_name, old_name, access_dist, k):

```

Comment:
I think we should provide some default values to guide the user as to possible sensible parameterisation of k; e.g.
`def Cumulative_Gaussian_access_score(df, new_name, old_name, access_dist=500, k=5)`
I'm not sure if k=5 is sensible for this formula for a distance of 500 metres, but if it might be handy having some kind of suggestion of an appropriate decay for 500 metres (ie. if we were to use it for our case study, what would be appropriate?)